components/Sidebar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+import useUser from 'hooks/useUser';
+import { clearToken } from 'helpers/authentication';
+
+export const sidebarTestIds = {
+  base: 'sidebar',
+  userName: 'sidebar__user-name',
+  button: 'sidebar__logout-button',
+};
+
+const Sidebar = ({ name }) => {
+  const { mutateUser } = useUser({
+    redirectTo: '/login',
+  });
+
+  const handleOnClick = async event => {
+    event.preventDefault();
+    clearToken();
+    mutateUser();
+  };
+
+  return (
+    <div
+      className="fixed bg-sidebar backdrop-blur-4xl right-0 w-[264px] pl-8 pt-8 pr-8 h-full"
+      data-test-id={sidebarTestIds.base}
+    >
+      <div className="text-white text-base-xxl font-extrabold mb-[22px]" data-test-id={sidebarTestIds.userName}>
+        {name}
+      </div>
+      <hr className="border-solid rounded-[0.5px] border-white/20 mb-[35px]" />
+      <Link href="/logout">
```

Comment:
Also, regarding the navigation item here should we adjust its side to include the space around it?

Current style | The design
--- | ---
![Screen Shot 2565-04-11 at 15 23 17](https://user-images.githubusercontent.com/1772999/162695855-d629c2cd-9095-4a6b-a4c4-416df38e247d.png) | ![Screen Shot 2565-04-11 at 15 23 50](https://user-images.githubusercontent.com/1772999/162695877-44cafa1d-6971-42c1-9d1c-05ed4e590de5.png)

Reply:
done in 2125544666199978509632db13f61e3cd5903fa6